mico-core/src/main/java/io/github/ust/mico/core/resource/ApplicationResource.java

```diff
@@ -233,38 +234,28 @@
         return ResponseEntity.noContent().build();
     }
 
-    @ApiOperation(value = "Adds a new association between a MicoApplication and a KafkaFaasConnector (MicoService). " +
+    @ApiOperation(value = "Adds or updates an association between a MicoApplication and a KafkaFaasConnector (MicoService). " +
```

Comment:
I don't like merging them like this. They clearly have a different semantik and should be differentiated in code also. Sharing code between these methods can also be done with a helper method that both of them call.